bundles/org.openhab.binding.dscalarm/src/main/resources/OH-INF/thing/zone.xml

```diff
@@ -13,6 +13,7 @@
 
 		<label>DSC Alarm Zone</label>
 		<description>Represents a physical device such as a door, window, or motion sensor.</description>
+		<semantic-equipment-tag>AlarmDevice</semantic-equipment-tag>
```

Comment:
Label and description seem off, not familiair with this binding, but looking at the label this might also qualify for a new tag: `AlarmZone`

Reply:
..